snooty/rstspec.toml

```diff
@@ -463,6 +463,43 @@ example = """.. code:: ${1:language}
    ${0:code content}
 """
 
+[directive.io-code-block]
+help = """A block of code and its corresponding output to show the user."""
+argument_type = "string"
+content_type = "block"
+options.caption = "string"
+options.copyable = "boolean"
+options.class = "string"
+options.source = "uri"
```

Comment:
Is `source` to be implemented in a future PR?

Reply:
Yep, once the LG changes for custom action buttons for the Code component are in prod, the frontend for this can then be implemented.